app/scripts/lib/setupSentry.js

```diff
@@ -321,17 +359,12 @@ function hideUrlIfNotInternal(url) {
  */
 export function beforeBreadcrumb(getState) {
   return (breadcrumb) => {
-    if (getState) {
-      const appState = getState();
-      if (
-        Object.values(appState).length &&
-        (!appState?.store?.metamask?.participateInMetaMetrics ||
-          !appState?.store?.metamask?.completedOnboarding ||
```

Comment:
We also need to account for the `!appState?.store?.metamask?.completedOnboarding` in the refactor. Perhaps by adding another new function like `getMetaMetricsEnabledFromAppState`

Reply:
What is the purpose of checking that property?

Reply:
Good catch, fixed in a8b1dafdd3

I left it inlined here for now since it's the only place we check it